pkg/pmem-device-manager/pmd-lvm.go

```diff
@@ -158,12 +158,18 @@ func (lvm *pmemLvm) DeleteDevice(name string, flush bool) error {
 	if err != nil {
 		return err
 	}
-	if err := ClearDevice(device, flush); err != nil {
+	if err = ClearDevice(device, flush); err != nil {
```

Comment:
`err :=` is better style because it makes this line independent of the preceding code.

Reply:
sorry for creating confusion, but the real home of the code you review, is under #408. In this PR I cherry-picked changes from #408 to cause combined state go through CI and thus demonstrate improvement that lvm.devices is not growing (which is otherwise not detectable from CI results). The meaning of this PR originally was only to demonstrate that lvm.devices is growing, so it's not meant for merging.

Reply:
@avalluri can you take that feedback into account in #408